apps/cli/package.json

```diff
@@ -32,7 +32,8 @@
 		"figlet": "^1.9.4",
 		"gradient-string": "^3.0.0",
 		"inquirer": "^12.5.0",
-		"ora": "^8.2.0"
+		"ora": "^8.2.0",
+		"terminal-link": "^5.0.0"
```

Comment:
if you add a new package in the apps/cli, make sure you also add it to the root, or else you'll get a "module not defined" at production